code/modules/xenoarcheaology/artifacts/artifact.dm

```diff
@@ -3,8 +3,9 @@
 	desc = "A large alien device."
 	icon = 'icons/obj/xenoarchaeology.dmi'
 	icon_state = "ano00"
+	density = TRUE
```

Comment:
Make sure the waterproof var is unset. Although I assume it may be.

Reply:
structures don't have waterproof var